roundabout-core/src/test/java/at.fhv.itm3.s2.roundabout.test/RoundaboutModelTest.java

```diff
@@ -0,0 +1,23 @@
+package at.fhv.itm3.s2.roundabout.test;
+
+import at.fhv.itm3.s2.roundabout.RoundaboutModel;
+import desmoj.core.simulator.Experiment;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RoundaboutModelTest {
+
+    @Test
+    public void getTimeBetweenCarArrivalsTest() {
```

Comment:
naming is not very meaningful
e.g.: getTimeBetweenCarArrivals_validTimeBoundary